apps/website/src/content/docs/tag.mdx

```diff
@@ -71,9 +80,8 @@ The scrollable container is much like the truncating container, except it allows
 
 <PropsTable path={'@itwin/itwinui-react/esm/core/Tag/TagContainer.d.ts'} />
 
-## Usage
+### Usage guidelines
```

Comment:
Is it alright to have two headings with the same name?

Reply:
Yeah it's fine. They are sub-headings